.prettierrc

```diff
@@ -0,0 +1,3 @@
+{
+  "singleQuote": true
```

Comment:
Generally speaking, this is now obsolete, and we prefer double-quoted strings.  Single-quote was from the historical practice of generating HTML in Javascript, since HTML attributes are double-quote delimited.

Reply:
Thanks for the info! I'll fix this at last since it may involve a lot of code change and might mess up the PR for review purpose :)

Reply:
Will resolve this in the next PR.